Mathlib/Tactic/ToAdditive/Frontend.lean

```diff
@@ -818,9 +818,10 @@ partial def transformDeclAux
   if isProtected (← getEnv) src then
     setEnv <| addProtected (← getEnv) tgt
   if let some matcherInfo ← getMatcherInfo? src then
-    -- Use
-    --   Match.addMatcherInfo tgt matcherInfo
-    -- once on lean 4.13.
+    /-
```

Comment:
```suggestion
    #adaptation_note
    /--
```
Or should we use some other marker, so that it will be recorded as tech debt?

Reply:
We can't used adaptation note in term mode.